src/main/scala/net/snowflake/spark/snowflake/SnowflakeTelemetry.scala

```diff
@@ -188,6 +188,31 @@ object SnowflakeTelemetry {
     )
   }
 
+  private[snowflake] def sendQueryResult(conn: Connection,
```

Comment:
is this sending the query result or the query status ?

Reply:
query status. I will change the function name as sendQueryStatus()